packages/features/ee/deployment/lib/getDeploymentKey.ts

```diff
@@ -1,9 +1,13 @@
 import type { PrismaClient } from "@calcom/prisma";
 
 export async function getDeploymentKey(prisma: PrismaClient) {
+  if (process.env.CALCOM_LICENSE_KEY) {
```

Comment:
It makes more sense to early return here if its found in .env. Saves a redundant db call

Reply:
I think the DB is the preferred way to check/save the key. The env one is only for backwards compatibility. If a self hosted uses the wizard to set update the key, then it will never reach it unless it manually removes it from env. Although it makes sense to alert the user that it might have differing keys in the DB/env.

Reply:
I get this - but to set the license key in the wizard it needs to be set in the .env. 

https://app.campsite.co/cal/posts/zxcq11vnt6tg#comment-l2toxbc9dr55